iothub/device/src/Pipeline/RetryDelegatingHandler.cs

```diff
@@ -60,14 +73,17 @@ public override async Task SendTelemetryAsync(TelemetryMessage message, Cancella
 
             try
             {
+                await VerifyIsOpenAsync(cancellationToken).ConfigureAwait(false);
+                using var operationCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, _cancelPendingOperationsCts.Token);
```

Comment:
This is the linked cancellation token that will be signaled for closure if `CloseAsync()` is called